### PR TITLE
records: refactor import article view

### PR DESCRIPTION
**Pull request by drjova**

### CODE REVIEW

File: backend/inspirehep/records/api/literature.py [hunk 1]
```diff
@@ -659,7 +657,7 @@ def merge_article_with_crossref_data(article):
 
     try:
         crossref_data = import_doi(doi)
-    except (ImportConnectionError, ImportParsingError):
+    except (requests.exceptions.RequestException, ImportParsingError):
```
Comment (MJedr) [inline backend/inspirehep/records/api/literature.py:660]: why we can't import it as `RequestException`?

**Review by MJedr — APPROVED** [review]

File: backend/inspirehep/records/api/literature.py (hunk 1 above)
Comment (drjova) [reply backend/inspirehep/records/api/literature.py:660]: indeed just because we are already importing the whole module

**Comment by drjova** [comment]
@michamos is it what we discussed?

**Review by michamos — APPROVED** [review]
looks good :+1:

**Pull request merged**